pages/updatePaymentMethod.js

```diff
@@ -231,8 +240,12 @@ class UpdatePaymentPage extends React.Component {
                           />
                         </Box>
                       )}
-                      {!showCreditCardForm && (
-                        <FormattedMessage id="success" defaultMessage="Your new card info has been added" />
+                      {!showCreditCardForm && error && error}
```

Comment:
Remove?

Reply:
It's meant to be there. It was `{!showCreditCardForm && error && (error)}` but the linter went a bit crazy. I've just tested it though and I can remove the second `&& error`